tests/unit/apps/test_auxiliary.py

```diff
@@ -280,6 +324,20 @@ def test_auxiliary_raise_halt_upgrade(status, config, model):
         app.generate_upgrade_plan(target)
 
 
+def test_auxiliary_no_suitable_channel(status, config, model):
+    # OPENSTACK_TO_TRACK_MAPPING can't find a track for rabbitmq, focal, zed.
+    target = OpenStackRelease("zed")
```

Comment:
tiny nit: the fact that `status["rabbitmq_server"].series == 'focal'` isn't immediately obvious. With tests generally needing to be geared for clarity and simplicity over compactness, I think explicitly setting series to focal would be nice here (even if unnecessary).